ceph-releases/jewel/ubuntu/14.04/daemon/common_functions.sh

```diff
@@ -122,7 +122,7 @@ function dev_part {
 }
 
 function osd_trying_to_determine_scenario {
-  ${OSD_DEVICE:-none}
+  : "${OSD_DEVICE:-none}"
```

Comment:
This line does not change the OSD_DEVICE value. If you want this var to get "none" value in case it is not set you should do `OSD_DEVICE="${OSD_DEVICE:-none}"`.